internal/controllers/reconciler_test.go

```diff
@@ -2336,7 +2336,18 @@ func (t *cryostatTestInput) expectCertificates() {
 	err := t.Client.Get(context.Background(), types.NamespacedName{Name: expectedSecret.Name, Namespace: expectedSecret.Namespace}, secret)
 	Expect(err).ToNot(HaveOccurred())
 	t.checkMetadata(secret, expectedSecret)
-	Expect(secret.StringData).To(Equal(secret.StringData))
+	Expect(secret.StringData).To(Equal(expectedSecret.StringData))
+
+	// Check CA Cert secrets in each target namespace
+	Expect(t.TargetNamespaces).ToNot(BeEmpty())
+	for _, ns := range t.TargetNamespaces {
+		namespaceSecret := t.NewCACertSecret(ns)
+		secret := &corev1.Secret{}
+		err := t.Client.Get(context.Background(), types.NamespacedName{Name: namespaceSecret.Name, Namespace: ns}, secret)
+		Expect(err).ToNot(HaveOccurred())
+		t.checkMetadata(secret, namespaceSecret)
+		Expect(secret.Data).To(Equal(namespaceSecret.Data))
```

Comment:
It would be good to add a regression test here to ensure that only the certificate gets copied but not the private key. To do this, you could add a dummy private key here: https://github.com/cryostatio/cryostat-operator/blob/e8e0bef0bd1c00bcc16d6a238a27c05263085a3b/internal/test/clients.go#L95

Something like: `corev1.TLSPrivateKeyKey: []byte(cert.Name + "-key"),` would work.

Back inside `expectCertificates`, we just need to exclude the install namespace from this check, since the original copy of the Secret will contain the private key. You can do that by adding a `if ns != t.Namespace {` block inside of the `for _, ns := range t.TargetNamespaces {` loop body.

I intended to mention this on my last review, but I must not have submitted the comment. Sorry about that.